Kentico.Kontent.Delivery/ContentItems/AssetElementValueConverter.cs

```diff
@@ -44,6 +44,8 @@ public AssetElementValueConverter(IOptionsMonitor<DeliveryOptions> options)
                 })
                 .Cast<IAsset>()
                 .ToList();
+
+            return Task.FromResult((object)assets);
```

Comment:
Is this a common practice @Sevitas @gormal @winklertomas?

Reply:
If we need to return `Task` then it's ok. But do we need to return `Task`?

Reply:
Yes, as it is implemented now, we unfortunately do because this method is implementation of method from `public interface IPropertyValueConverter<in T>`.

To be honest, best solution would be be definitely to have lets say synchronous and asynchronous version of this method, but that would need more internal changes. We can of course discuss this.